profiles/package.mask

```diff
@@ -30,6 +30,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Alexander Puck Neuwirth <alexander@neuwirth-informatik.de> (2024-12-04)
+# Depends on removed version of dev-scheme/guile:12
```

Comment:
Note that guile is just slotted now, so removed is accurate but not quite so helpful -- it's more that the package needs porting to the slotted guile eclass, no?

Reply:
It depends on `dev-scheme/guile:12` and `eix` shows me 1.8 2.2 and 3.0 as slots.

Reply:
guile 1.8.8-r5 was dropped in 7df4aaeb4d02b0c7e225c8194a5962c5d46507ee in gentoo.git where it had its slot as 12. I have no idea why it had that...

Reply:
I see they (new slots) all were slot 12 previously.

Reply:
Ok, I will give porting it a quick try.

Reply:
Only `1-8` passes tests, which is not surprising, given the code is from 2008 or so.

Reply:
Thank you!